views/components/config_card.erb

```diff
@@ -0,0 +1,95 @@
+<%# locals: (title:, config:, extra_class:, name:, errors:) %>
+
+<div class="bg-white rounded-lg shadow overflow-hidden mb-6 <%= extra_class %>">
+  <div class="px-6 py-4 border-b border-gray-200">
+    <h3 class="text-lg font-medium text-gray-900"><%= title %></h3>
+  </div>
+
+  <div class="divide-y divide-gray-200 w-full max-h-64 overflow-y-auto">
+    <% config.each_with_index do |(key, value), index| %>
```

Comment:
```suggestion
    <% config.each_with_index do |(key, value), index| %>
      <% error = errors[key] %>
```

And use `error` instead of `errors[key]` inside the block.